website/docs/Pipelines/ImputationBeagle_Pipeline/README.md

```diff
@@ -41,43 +41,47 @@ For examples of how to specify each input in a configuration file, as well as cl
 | sample_chunk_size           | Number of samples to chunk by when processing (default: 1,000)                                           | Int              |                                                                                                      
 | multi_sample_vcf            | Merged VCF containing multiple samples; can also use an array of individual VCFs.                        | File             |                                                                                                     
 | ref_dict                    | Reference dictionary.                                                                                    | File             |                                                                                                     
-| contigs                     | Array of strings defining which contigs (chromosomes) should be used for the reference panel.            | Array of strings |                                                                                         
+| contigs                     | Array of allowed contigs/chromosomes to process                                                          | Array of strings |                                                                                         
```

Comment:
thank you for updating this and adding `CalculateContigsToProcess` below! sorry I missed this